rust/main/agents/relayer/src/test_utils/request.rs

```diff
@@ -10,6 +10,8 @@ pub async fn parse_body_to_json<T: DeserializeOwned>(body: Body) -> T {
         .to_bytes()
         .into_iter()
         .collect();
+
+    eprintln!("{:?}", String::from_utf8(resp_body.clone()));
```

Comment:
I wonder if we want to keep it or it was added for debug purposes during implementation.